code/modules/reagents/chemistry/holder.dm

```diff
@@ -194,8 +218,7 @@ var/const/INGEST = 2
 
 /datum/reagents/proc/metabolize(mob/living/M)
 	if(M)
-		chem_temp = M.bodytemperature
-		handle_reactions()
+		set_reagent_temp(M.bodytemperature)
```

Comment:
maybe change to adjust the temp over time instead of setting it instantly? Needs testing

Reply:
I think this is probably ok, for now, but may need to be changed later, like post crit refactor.